### PR TITLE
IP Allow follow-ups

### PR DESCRIPTION
**Pull request by danieltprice**
- Add CLI and API commands for IP Allow
- Add "update configuration" instructions
- Add "remove configuration" instructions

Mange projects:
https://neon-next-git-dprice-ip-allow-followups-neondatabase.vercel.app/docs/manage/projects#configure-ip-allow

Neon CLI ip-allow command:
https://neon-next-git-dprice-ip-allow-followups-neondatabase.vercel.app/docs/reference/cli-ip-allow

### CODE REVIEW

File: content/docs/manage/projects.md [hunk 1]
```diff
@@ -238,6 +302,65 @@ A combined example using all three options above, specified as a comma-separated
 
 This list combines individual IP addresses, a range of IP addresses, and a CIDR block. It illustrates how different types of IP specifications can be used together in a single allowlist configuration, offering a flexible approach to access control.
 
+#### Update an IP Allow configuration
+
+You can update your IP Allow configuration via the Neon Console or API as described in [Configure IP Allow](#configure-ip-allow). WReplace the current configuration with the new configuration. For example, if your IP Allow configuration currently allows access from IP address `192.168.1.1`, and you want to extend access to IP address `192.168.1.2`, specify both addresses in your new configuration: `192.168.1.15, 192.168.1.15`. You cannot append values to an existing configuration. You can only replace an existing configuration with a new one.
```
[the remark this answers is not in the document]
Comment (Shridhad) [reply content/docs/manage/projects.md:307]: > For example, if your IP Allow configuration currently allows access from IP address `192.168.1.1`, and you want to extend access to IP address `192.168.1.2`, specify both addresses in your new configuration: `192.168.1.15, 192.168.1.15``

Looks like the Ip addresses are not in sync.

File: content/docs/reference/cli-ip-allow.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+```
+
+#### Options
+
+In addition to the Neon CLI [global options](/docs/reference/neon-cli#global-options), the `add` subcommand supports these options:
+
+| Option        | Description | Type   | Required  |
+| ------------- | ----------- | ------ | :------: |
+| `--context-file` | [Context file](/docs/reference/cli-set-context#using-a-named-context-file) path and file name | string | |
+| `--project-id`  | Project ID  | string | Only if your Neon account has more than one project |
```
Comment (Shridhad) [inline content/docs/reference/cli-ip-allow.md:80]: `--primary-only` option should also show here

File: content/docs/reference/cli-ip-allow.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+<CodeBlock shouldWrap>
+
+```bash
+neonctl ip-allow reset 192.168.1.1
+```
+
+</CodeBlock>
+
+<NeedHelp/>
```
Comment (Shridhad) [inline content/docs/reference/cli-ip-allow.md:150]: All of the examples on `ip-allow` are missing the mention `--project-id` option/argument. The command will fail without `--project-id`. Users either have to specify the option or add it to context file.

File: content/docs/manage/projects.md [hunk 4]
```diff
@@ -188,24 +188,88 @@ After enabling logical replication, the next steps involve creating publications
...
+├─────────────────────┼─────────────────────┼──────────────┼─────────────────────┤
+│ wispy-haze-26469780 │ wispy-haze-26469780 │ 192.168.1.1  │ false               │
+│                     │                     │ 192.168.1.2  │                     │
+└─────────────────────┴─────────────────────┴──────────────┴─────────────────────┘
+```
+
+To apply an IP allowlist to the primary branch only, use the [Neon CLI projects command](/docs/reference/cli-projects) with the `--ip-primary-only` option:
+
+```bash
+neonctl projects update wispy-haze-26469780 --ip-primary-only
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/manage/projects.md:234]: Fixed.

File: content/docs/manage/projects.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/manage/projects.md:307]: Fixed.

File: content/docs/reference/cli-ip-allow.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/reference/cli-ip-allow.md:80]: fixed.

File: content/docs/reference/cli-ip-allow.md (hunk 3 above)
Comment (danieltprice) [reply content/docs/reference/cli-ip-allow.md:150]: Added

File: content/docs/reference/cli-ip-allow.md [hunk 5]
```diff
@@ -78,13 +78,14 @@ In addition to the Neon CLI [global options](/docs/reference/neon-cli#global-opt
 | ------------- | ----------- | ------ | :------: |
 | `--context-file` | [Context file](/docs/reference/cli-set-context#using-a-named-context-file) path and file name | string | |
 | `--project-id`  | Project ID  | string | Only if your Neon account has more than one project |
+| `--primary-branch`  | If true, the list will be applied only to the primary branch. Use `--primary-branch false` to remove this setting. | string | |
```
Comment (Shridhad) [inline content/docs/reference/cli-ip-allow.md:81]: This should be `--primary-only`

**Review by Shridhad — APPROVED** [review]
just one more comment

File: content/docs/reference/cli-ip-allow.md (hunk 5 above)
Comment (danieltprice) [reply content/docs/reference/cli-ip-allow.md:81]: fixed

**Pull request merged**